cves/CVE-2011-3944.yml

```diff
@@ -355,7 +355,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: https://cwe.mitre.org/data/definitions/434.html
 CWE_note:
 nickname_instructions: |
```

Comment:
No nickname?? This was a perfect opportunity for something cool like `Smack Injection` or `Unwanted Smack`

Reply:
SmackerSmacks :P

Reply:
YEAH good name